cmd/tether/attach_test.go

```diff
@@ -141,7 +141,7 @@ func mockBackChannel(ctx context.Context) (net.Conn, error) {
 			// FIXME: need to implement timeout of purging hangs with no content
 			// on the pipe
 			// serial.PurgeIncoming(ctx, conn)
-			err := serial.HandshakeClient(ctx, conn)
+			err := serial.HandshakeClient(ctx, conn, true)
```

Comment:
[cosmetic] how about adding a new one named HandshakeClientWithCheck and calling it when needed instead of adding another parameter to the function?